python/strawReco.py

```
@@ -0,0 +1,152 @@
+    prop_options = acts.PropagatorPlainOptions(geo_ctx, mag_ctx)
+    prop_options.pathLimit = 100000.0  # 100 meters
+
+    distMin = 99999
+    hitID = -1
+
+    for t_idx, track in enumerate(output_tracks):
+        if not track.hasReferenceSurface():
+            continue
+        start_params = track.parameters()
+
+        for s_idx, sbt_hit in enumerate(sbt_digis):
```

Ruff (0.15.12)

[warning] Loop control variable `s_idx` not used within loop body

Rename unused `s_idx` to `_s_idx`

(B007)